src/index.js

```diff
@@ -14,9 +14,14 @@ app.use(bodyParser.text());
 app.use(bodyParser.json({ type: 'application/json' }));
 app.use('/', router);
 
+app.use('/', router);
+
```

Comment:
Can we remove one of these lines: `app.use('/',  router);` ?
It seems that it's duplicated.